Code/Framework/AzCore/AzCore/DOM/DomPath.h

```diff
@@ -68,7 +68,7 @@ namespace AZ::Dom
     class Path final
     {
     public:
-        AZ_TYPE_INFO(AZ::Dom::Path, "{C0081C45-F15D-4F46-9680-19535D33C312}")
+        AZ_RTTI(AZ::Dom::Path, "{C0081C45-F15D-4F46-9680-19535D33C312}");
```

Comment:
AZ_RTTI is not needed for reflection.
It is only needed if the Path class is base class of some other class that needs TypeInfo.
Also for AZ_RTTI, the fully qualified name should not be used, but instead just the name of "Path"
```suggestion
        AZ_TYPE_INFO(Path, "{C0081C45-F15D-4F46-9680-19535D33C312}");
```